include/deal.II/fe/fe_values.h

```diff
@@ -292,6 +316,13 @@ namespace FEValuesViews
                               std::vector<typename ProductType<value_type,typename InputVector::value_type>::type> &values) const;
 
     /**
+     *
+     */
+    template <class InputVector>
+    void get_function_values_from_local_dof_values (const InputVector &dof_values,
+                                                    std::vector<typename OutputType<typename InputVector::value_type>::value_type> &values) const;
```

Comment:
The documentation is missing here.

Why use `OutputType` here when the previous function just inlines the whole expression?